gui/default/syncthing/settings/settingsModalView.html

```diff
@@ -172,6 +172,13 @@
               </div>
             </div>
             <div class="col-md-6">
+              <div ng-if="tmpGUI.address.startsWith('/')" class="form-group" ng-class="{'has-error': settingsEditor.UnixPerms.$invalid && settingsEditor.UnixPerms.$dirty}">
```

Comment:
unix:// and unixs:// are also valid prefixes.

Reply:
I can't get that to work, it tries using TCP. Testing through the UI also won't work, because the Regex that verifies the address does not allow such a prefix, so you cannot save the page. In the code, it only checks for '/' to determine whether it's a UNIX socket (unless the environment variable is used, in which case it resolves then checks for the `unix` prefix).

In order to comply, I would need to alter how the code understands `Address`. If you want me to do that, let me know and I will do it.

Reply:
I would add the code to check for prefix, even if it's not possible, so that when the regexp is modified, it's not overlooked.

Also, what happens if you put unix:// in the config by hand and then open the UI? Does it get removed? If not, then perhaps we should adjust the regexp.

Also, you suggest that it tries unix:// as TCP, yet we have code that explicitly handles unix://, so I am surprised it does not work.

Reply:
Fair point about adding it.
If it is using config settings for address, then it only checks `/`.
If it is using environment variables, it checks for `unix` in the scheme.